plugin/ui/completion.py

```diff
@@ -78,15 +55,11 @@ def completions(self, value: List[CopilotPayloadCompletion]) -> None:
     @property
     def completion_style(self) -> str:
         """The completion style."""
-        return get_copilot_view_setting(self.view, "completion_style", self.COMPLETION_STYLES[0])
+        return get_copilot_view_setting(self.view, "completion_style", "")
```

Comment:
If it will break, then let it break (exception in `completion_style_type`). The user should fix it.